app/assets/stylesheets/partials/_navbar.sass

```diff
@@ -2,14 +2,14 @@ nav#navigation
   @extend .navbar, .navbar-inverse, .navbar-static-top
 
   a.nav-dropdown-img-toggle
-    @extend .dropdown-toggle, .hidden-xs, .btn, .navbar-btn
```

Comment:
So just for a quick reference: with dropping these classes, you end up getting an extra empty dropdown:
![screen shot 2017-04-23 at 13 02 03](https://cloud.githubusercontent.com/assets/3491815/25312981/fbe1067c-2825-11e7-97c1-3bab4ff0a6d0.png)

vs. the *"old"*:
![screen shot 2017-04-23 at 13 07 56](https://cloud.githubusercontent.com/assets/3491815/25312989/0bd731be-2826-11e7-9d5b-729e1a9a8728.png)